php/.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
```

Comment:
Not sure if any of these files are like node_modules or a package.lock file, but anything like that does not need to be included. I actually need to update the node and next example to remove some of the bloat.

Reply:
This is similar to prettier - just some style configs. I added it to gitignore